client/src/components/productOverview/StyleRow.jsx

```diff
@@ -3,7 +3,7 @@ import StyleThumbnail from './StyleThumbnail.jsx';
 
 const StyleRow = (props) => (
```

Comment:
You can use destructuring like you did in ProductInfo.jsx.

Reply:
Yes, good call. Consistency is probably the way to go. Thanks